cmd/web.go

```diff
@@ -71,6 +73,24 @@ func runHTTPRedirector() {
 	}
 }
 
+func runLetsEncrypt(listenAddr, domain string, m http.Handler) error {
+	certManager := autocert.Manager{
+		Prompt:     autocert.AcceptTOS,
+		HostPolicy: autocert.HostWhitelist(domain),
+		Cache:      autocert.DirCache("https"),
```

Comment:
I think this directory should come from `app.ini` file

Reply:
Also I think there should be option to specify `Email` that will be used by letsencrypt registration

Reply:
updated with your changes